app/controllers/api/v2/courses_controller.rb

```diff
@@ -30,6 +30,14 @@ def build_new
           class: CourseSerializersService.new.execute,
           include: [:subjects, :sites, :accrediting_provider, :provider, provider: [:sites]],
         )
+        subjects = json_data[:data][:meta][:edit_options][:subjects]
+
+
+        if !@current_user.admin? && subjects_include_pe(subjects)
```

Comment:
This extra operation of checking that PE exists in the subjects with `subjects_include_pe`, is a little bit expensive (computing-wise) and I'm not sure it really adds a lot. I'd suggest just taking it out and letting the reject run since it'll be a no-op if it's not there anyway.

Reply:
Done. Thanks